### ISSUE TITLE
Is there a Windows implementation?

### ISSUE THREAD

**Issue by juebanlin** [MediaBrowser/Emby.IsoMounting#5]
Ubuntu cannot use dual graphics cards. In the case of virtual machines, only windows compatible graphics cards are directly connected.Hope to solve the mount problem of ISO under the windows platform。

**Comment by LukePulverenti** [comment]
We used to using software called Pismo but over time as things changed it had to be rewritten. At this point there are really no plans to resurrect it ourselves as I just don't think there's enough of an audience for iso files anymore. It could be done via plugin by a community member though.

**Comment by juebanlin** [comment]
@LukePulverenti 

Is there a community plug-in that implements this function? If I do it myself, how do I add it to emby server, and compile it into DLL for a directory?

**Comment by LukePulverenti** [comment]
For Windows, no, it will need to be developed.

**Comment by LukePulverenti** [comment]
What is the approach that you wish to use for windows?

**Comment by juebanlin** [comment]
@LukePulverenti 
Is this project compiled into DLL and put into a directory? I want to try it.
 sample：
#挂载iso
PS F:\> Mount-DiskImage F:\ubuntu-16.04.3-desktop-amd64.iso

#获取磁盘镜像信息
PS F:\> Get-DiskImage -ImagePath "F:\ubuntu-16.04.3-desktop-amd64.iso"


Attached          : True
BlockSize         : 0
DevicePath        : \\.\CDROM0
FileSize          : 1587609600
ImagePath         : F:\ubuntu-16.04.3-desktop-amd64.iso
LogicalSectorSize : 2048
Number            : 0
Size              : 1587609600
StorageType       : 1
PSComputerName    :


#根据设备路径获取挂载点信息
PS F:\> Get-DiskImage -DevicePath \\.\CDROM0 | Get-Volume

DriveLetter FriendlyName     FileSystemType DriveType HealthStatus OperationalStatus SizeRemaining    Size
----------- ------------     -------------- --------- ------------ ----------------- -------------    ----
D           Ubuntu 16.04.3 L Unknown        CD-ROM    Healthy      OK                          0 B 1.48 GB


#根据iso路径获取挂载点信息
PS F:\> Get-DiskImage -ImagePath "F:\ubuntu-16.04.3-desktop-amd64.iso" | Get-Volume

DriveLetter FriendlyName     FileSystemType DriveType HealthStatus OperationalStatus SizeRemaining    Size
----------- ------------     -------------- --------- ------------ ----------------- -------------    ----
D           Ubuntu 16.04.3 L Unknown        CD-ROM    Healthy      OK                          0 B 1.48 GB

#卸载ISO
PS F:\> Dismount-DiskImage -ImagePath "F:\ubuntu-16.04.3-desktop-amd64.iso"

#卸载ISO
PS C:\>Dismount-DiskImage -DevicePath \\.\CDROM0

**Comment by juebanlin** [comment]
@LukePulverenti 
You can tell me how this plug-in project is deployed to windows. Compiled for the DLL file to burn into the emby installation directory?

**Comment by juebanlin** [comment]
@LukePulverenti 
Can you add the implementation on windows? I tested that Pismo can be implemented.

This is my example: https://github.com/juebanlin/emby.WinIsoMount
thanks。

**Comment by juebanlin** [comment]
> 
> 
> I will not be able to spend time on this, sorry. The audience for it is too small. But you could handle it all on your own, just like this plugin is.

Thank you. I understand very well. I tried to add my own plug-in, but the current version does not work properly with the plug-in I wrote myself. The reason is that emby on Windows did not call my plug-in. I checked the source code and found that the calling logic was not open source. So, I guess, is it that the plug-in using ISO mount is closed on Windows? Can the next version help open this part of logic? Thank you。
